src/main/kotlin/org/turfkt/Measurement.kt

```diff
@@ -0,0 +1,127 @@
+package org.turfkt
+
+import geojsonkt.Geometry
+import geojsonkt.Feature
+import geojsonkt.FeatureCollection
+import geojsonkt.Polygon
+import geojsonkt.MultiPolygon
+import geojsonkt.Point
+import geojsonkt.MultiPoint
+import geojsonkt.LineString
+import geojsonkt.MultiLineString
+import geojsonkt.Position
+
+import kotlin.math.sin
+import kotlin.math.abs
+
+// For testing purposes
+//val EARTHRADIUS = 6378137
+
+private fun calculateArea( geom: Geometry): Double{
+    var total = 0.0
+    //var i: Int
+    when (geom) {
+        is Polygon -> { 
+            //val size = geom.coordinates.size
+            return polygonArea(geom.coordinates)
+        }
+        is MultiPolygon -> {
+            //val size = geom.coordinates.size
+            for (i in 0..(geom.coordinates.size - 1)) {
```

Comment:
Use `for(i in 0 until geom.coordinates.size`) or `for(i in geom.coordinates.indices)`